docs/install_guides/uninstalling-openvino.md

```diff
@@ -113,3 +115,29 @@ Finally, complete the procedure with clicking on **Modify** and then selecting *
     :align: center
 
 @endsphinxdirective
+
+## Uninstalling OpenVINO 2022.1.1
+
+If you have installed OpenVINO Runtime 2022.1.1 from archive files, you can uninstall it by deleting the archive files and the extracted folders.
+
+@sphinxdirective
+.. tab:: Windows
+
+  You can choose either of the following methods:
+
+  * Use Windows Explorer to remove the files.
+  * Open a Command Prompt and run:
+    
+    .. code-block:: sh
+  
+      rmdir /s <extracted_folder>
+      del <path_to_archive>
+
+    
+.. tab:: Linux & macOS
+  
+  .. code-block:: sh
+  
+    rm -r <extracted_folder> && rm <path_to_archive>
```

Comment:
If we have created `openvino_2022` symlink, should we unlink it first? Otherwise, it will be broken I suppose..

Symlink was the optional step in Install openvino from archives.

Reply:
@ilya-lavrenov You mean using this command `rm /home/<USER>/intel/openvino_2022`? Then how about uninstalling on Windows?

Reply:
right, `rm` or `unlink` commands can be used on Linux.

>  Then how about uninstalling on Windows?

Do we recommend to create symlinks? If yes, we can use this guide https://www.computerhope.com/mklink.htm#:~:text=To%20delete%20a%20symbolic%20link,link%20use%20the%20del%20command. it shows that we can use `mklink` and `del` to create and delete symlinks accordingly (have not tested locally)

Reply:
Hi @ilya-lavrenov , @zKulesza can either of you provide the full form the commands to delete symlinks on Windows? Thanks a lot :)